### DISCUSSION TITLE
Floating Firefox Notification

### DISCUSSION THREAD

**Discussion by companion256** [i3/i3#5952] [Q&A] [answered]
I'm using i3 Manjaro. Firefox sends notifications and instead of floating window in the top right corner, it creates an entire window. 

I'm unable to locate PID, since I believe it's a Firefox subprocess. How can I make the notifications float again?

I may remove the notifications.

**Comment by companion256** [comment, marked as answer]
took me awhile though it looks like it's a window_role alert

`for_window [class="Firefox" window_role="alert"] floating enable`

**Reply by AntiCompositeNumber** [reply]
for some reason, I had to use `firefox` instead of `Firefox`. So if it doesn't work try that.

**Comment by ayjayt** [comment]
hey @companion256 did this _just_ start for you with an update? Because I didn't have this problem till I updated yesterday and I see your post is recent. Also, did you place that in your i3 config?

**Reply by companion256** [reply]
Yep! I put that in my i3 config. It's a good quick fix because there is less of an annoyance. 

However, with multiple notification, they stack over each other.

**Reply by ayjayt** [reply]
Are they also drawing focus for you?

**Reply by companion256** [reply]
I'm unsure actually :thinking: maybe

**Reply by AntiCompositeNumber** [reply]
It is, yes.

**Reply by ayjayt** [reply]
```
for_window [class="firefox" window_role="alert"] floating enable
no_focus [window_role="alert"]
```
